src/js/components/Chart/stories/Dash.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Chart } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const Example = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" pad="large" gap="large">
+      <Chart type="line" dash values={[20, 30, 15]} />
+      <Chart type="line" dash round values={[20, 30, 15]} />
+      <Chart type="line" dash thickness="xsmall" values={[20, 30, 15]} />
+      <Chart type="line" dash round thickness="xsmall" values={[20, 30, 15]} />
+      <Chart type="line" dash="8 2" values={[20, 30, 15]} />
```

Comment:
If you are not familiar with the CSS styling for dasharray, It is not clear what the `dash` values of "8 2" mean, we need to elaborate on that on doc.js.

Reply:
Simplified to remove the string capability for now.